source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py

```diff
@@ -312,6 +312,15 @@ def get_ridesharing_journeys(self, obj):
             return None
         return JourneySerializer(obj.ridesharing_journeys, display_none=False, many=True).data
 
+    cycle_lane_length = jsonschema.MethodField(
+        schema_type=int, display_none=False, description='Length of the section on cycle lanes'
+    )
+
+    def get_cycle_lane_length(self, obj):
+        if not hasattr(obj, 'cycle_lane_length') or not obj.cycle_lane_length:
```

Comment:
minor: Don't you want to output `0` when you have the information and you know it's 0 ?
With this it's not possible to differentiate `0` vs. `no information available/accurate`.
Don't know if it's easily doable to differentiate though, depending on the outputs of connectors (and what protobuf allows).

Reply:
That's the behavior I wanted. Did it with `PbIntField(display_none=False)`